docs/en/api/POSIX-API-local-cache.md

```diff
@@ -0,0 +1,438 @@
+```console
+$ bin/alluxio-fuse mount ufs_path mount_point -o attr_timeout=600 -o entry_timeout=600 
+```
+
+Recommend to set the timeout values based on the following factors:
+- Memory resources. The longer the timeout, the more metadata may be cached by kernel which contributes to higher memory consumption.
+  One can pre-decide how much memory to allocate to metadata kernel cache first.
+  Monitor the actual memory consumption while setting a large enough timeout value.
+  Then decide the timeout value suitable for the target memory usage.
+- Dataset in-use time. If the timeout value is bigger than the whole dataset in-use time and there are enough available memory resources,
+  cache invalidation and refresh will not be triggered, thus the highest cache-hit ratio and best performance can be achieved.
+- Dataset size. Kernel metadata cache for a single file takes around 300 bytes (up to 1KB), 3GB (up to 10GB) for 10 million files.
+  If the memory space needed for caching the metadata of the whole dataset is much smaller than the available memory resources,
+  recommend setting the timeout to your dataset in-use time. Otherwise, you may need to trade-off between memory consumption and cache-hit ratio.
+
+Note that, even a short period (e.g. `timeout=60` or `timeout=600`) of kernel metadata cache may significantly improve the overall metadata read performance and/or data read performance.
+Test against your common workloads to find out the optimal value.
+
+{% endnavtab %}
+{% navtab Userspace Metadata Cache Configuration %}
+
+Userspace metadata cache can be enabled via
+```console
+$ bin/alluxio-fuse mount ufs_path mount_point -o metadata_cache_size=<size> -o metqadata_cache_expire=<timeout>
+```
+`metadata_cache_size` (Default = `0` which means disabled): Maximum number of entries in the metadata cache. Each 1000 entries cause about 2MB memory.
+`metadata_cache_expire` (Default = `10min`): Specify expire time for entries in the metadata cache
+
+The metadata is cached in the AlluxioFuse Java process heap so make sure `metadata_cache_size * 2KB * 2 < AlluxioFuse process maximum memory allocated`.
+For example, if AlluxioFuse is launched with `-Xmx=16GB` and metadata cache can use up to 8GB memory, then `metadata_cache_size` should be smaller than 4 million.
+
+{% endnavtab %}
+{% endnavtabs %}
+
+#### Data Cache
+
+{% navtabs dataCache %}
+{% navtab Kernel Data Cache Configuration %}
+
+FUSE has the following I/O modes controlling whether data will be cached and the cache invalidation policy:
+- `direct_io`: disables the kernel data cache. Does not support in AlluxioFuse with libfuse3 yet.
+- `kernel_cache`: always cache data in kernel and no cache invalidation is happening. This should only be enabled on filesystem where the file data is never changed externally (not through the current FUSE mount point)
+- `auto_cache`: cache data in kernel and invalidate cache if the modification time or the size of the file has changed
```

Comment:
These three modes seem to be "kernel cache" mode.

What about "user space data cache"?

So you can see, the "-o data_cache=<local_cache_directory> -o data_cache_size=<size>" option below, is it one of the mode?
But it is "user space cache".

Reply:
Yeah it's under the section
```
{% navtabs dataCache %}
{% navtab Kernel Data Cache Configuration %}
```
this is a way to only show the title, after you click the title, you can see the information under this tile

For user side we have
```
{% navtab Userspace Data Cache Configuration %}
```